RELEASE.md

```diff
@@ -0,0 +1,36 @@
+# Releasing connect-go
+
+This document outlines how to create a release of connect-go.
+
+1. Clone the repo, ensuring you have the latest main.
+
+2. On a new branch, open [connect.go](connect.go) and change the `Version` constant. Do not just remove the `-dev` suffix: look at the release history and the unreleased commits to choose a new semantic version number.
+
+   ```patch
+   -const Version = "1.14.0-dev"
+   +const Version = "1.14.0"
+   ```
+
+4. Open a PR titled "Prepare for vX.Y.Z" ([Example PR #661](https://github.com/connectrpc/connect-go/pull/661)). Once it's reviewed and CI passes, merge it. *Make sure no new commits are merged until the release is complete.*
+
+5. Using the Github UI, create a new release.
+    - Under “Choose a tag”, type in “vX.Y.Z” to create a new tag for the release upon publish.
+    - Target the main branch.
+    - Title the Release “vX.Y.Z”.
+    - Click “set as latest release”.
+    - Set the last version as the “Previous tag”.
+    - Click “Generate release notes” to autogenerate release notes, sort them into ### Enhancements and ### Bugfixes, and edit the PR titles to be meaningful to end users. Feel free to collect multiple small changes to docs or Github config into one line, but try to tag every contributor. Make especially sure to credit new external contributors!
+
+6. Publish the release.
+
+7. Take the newly created release, click on the button to edit the release, and then update the release. See this [issue](https://github.com/orgs/community/discussions/8226) for guidelines.
+
```

Comment:
Let's move this to the end and frame it more like, "Check the releases page to see if releases are out of order. If they are, first take the release you just did, click on the button to edit the release, and then update the release. If that doesn't work, contact GitHub support (link) and explain the problem to them. They can trigger a re-index."

Reply:
I linked to the disucssions. Not sure what link is for github support if that is what you meant?